copilot/fastapi/app/config.py

```
@@ -0,0 +1,91 @@
+    tvp_branch: str = os.getenv("TVP_BRANCH", "main")
+    
+    # Environment
+    environment: str = os.getenv("ENVIRONMENT", "development")
+    
+    # Security
+    verify_ssl: bool = os.getenv("VERIFY_SSL", "true").lower() != "false"
+    
+    class Config:
+        env_file = ".env"
+        
+    def __init__(self, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)